CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.23.8] - 2022-08-31
```

Comment:
Need the link to [0.23.8] defined at the bottom

Reply:
Thanks, updated.